frontend/packages/console-shared/src/components/dashboard/launcher-card/LauncherBody.tsx

```diff
@@ -0,0 +1,9 @@
+import * as React from 'react';
+
+import './launcher-card.scss';
+
+const LauncherBody: React.FC = ({ children }) => (
+  <div className="co-dashboard-card__body--top-margin co-launcher-card__body">{children}</div>
```

Comment:
I guess this is how we've done it for other dashboard cards, but it feels like overkill to me to create a component just for one div

Reply:
the reasoning here is that the <Foo>Body will be used across different dashboards and Im trying to save the other devs from always have to define the style in their div and also when we decide to change the <Foo>Body style, we will change it in one place, the downside is, as you wrote, having a component which is just a div. I see that @rhamilto is doing some changes related to card bodies in https://github.com/openshift/console/pull/3031 so maybe we will change this too